js/background.js

```diff
@@ -638,23 +638,22 @@
     window.doUpdateGroup = async (groupId, groupName, members, avatar) => {
       const ourKey = textsecure.storage.user.getNumber();
 
-      const ev = new Event('message');
-      ev.confirm = () => {};
-
-      ev.data = {
-        source: ourKey,
-        timestamp: Date.now(),
-        message: {
-          group: {
-            id: groupId,
-            type: textsecure.protobuf.GroupContext.Type.UPDATE,
-            name: groupName,
-            members,
-            avatar: null, // TODO
-          },
+      const ev = {
+        groupDetails: {
+          id: groupId,
+          name: groupName,
+          members,
+          recipients: members,
+          active: true,
+          expireTimer: 0,
```

Comment:
is this time until key rotation?

Reply:
No, it is `expireTimer` on the conversation (for disappearing messages?) that underlying functions expect.